src/libsync/propagatedownload.h

```diff
@@ -260,5 +262,9 @@ private slots:
     QElapsedTimer _stopwatch;
 
     PropagateDownloadEncrypted *_downloadEncryptedHelper = nullptr;
+
+    std::filesystem::perms _parentPermissions;
+    std::filesystem::path _parentFolderPath;
```

Comment:
@mgallien I guess `_parentFolderPath` can be just `_parentPath`, since you have `_parentPermissions` not `_parentFolderPermissions` and a parent path can only be a folder.